sdcm/sct_events.py

```diff
@@ -877,17 +883,18 @@ def __eq__(self, other):
         return True
 
 
-class TestKiller(Process):
+class TestKiller(multiprocessing.Process):
```

Comment:
this class is not actually used it can be deleted AFAIK

Reply:
It used in unit tests. Probably it can be moved to `unit_tests/test_events.py`

Reply:
test can be deleted too, 
those take are marked as skipped, we were saving them to maybe we would use it.
and in the end we didn't use that class for killing the test, we did it with a signal in a different way

it can be safely deleted

Reply:
Done